lang/English/modules/settings.php

```diff
@@ -45,7 +45,7 @@
 define('feed_url', "Feed URL");
 define('feed_url_info', "GrayCube.com is sharing a LGSL feed on the URL:<br><b>http://www.greycube.co.uk/lgsl/feed/lgsl_files/lgsl_feed.php</b>");
 define('charset', "Character Encoding");
-define('charset_info', "UTF8, ISO, ASCII, etc... Left it blank to use ISO encoding.");
+define('charset_info', "UTF8, ISO, ASCII, etc... Leave it blank to use ISO encoding.");
```

Comment:
It's used on the panel settings page when hovering over the Character Encoding tooltip.

'Leave it blank' seems more natural than 'left it blank' - but if the word left needs to be used, then I'd probably phrase it, 'If left blank ISO encoding will be used.'